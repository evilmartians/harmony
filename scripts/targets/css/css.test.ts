import { expectBuildToMatchSnapshot } from "../../../tests/utils.ts";
import css from "./css.config.ts";

Deno.test("build css target", async (t) => {
  await expectBuildToMatchSnapshot(t, css);
});
