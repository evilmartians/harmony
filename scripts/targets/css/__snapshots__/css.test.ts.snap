export const snapshot = {};

snapshot[`index.css 1`] = `
'@import "./red.css";
@import "./orange.css";
'
`;

snapshot[`red.css 1`] = `
":root {
  --red-50: #fefafa;
  --red-500: #fc8083;
}

@supports (color: oklch(0% 0 0)) {
  @media (color-gamut: p3) {
    :root {
      --red-50: oklch(0.988281 0.0046875 20);
      --red-500: oklch(0.742188 0.151562 20);
    }
  }
}
"
`;

snapshot[`orange.css 1`] = `
":root {
  --orange-100: #fff1eb;
}

@supports (color: oklch(0% 0 0)) {
  @media (color-gamut: p3) {
    :root {
      --orange-100: oklch(0.966797 0.0171875 43.3333);
    }
  }
}
"
`;
